### PR TITLE
Testnet incident release

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

Release note for updated mac node.

## Changes

Added release notes for testnet, mainnet. Updated downloads pages.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

### CODE REVIEW

File: source/mainnet/net/resources/release-notes-mainnet.rst [hunk 1]
```diff
@@ -11,6 +11,13 @@ Release notes - Mainnet
 
    To subscribe to updates on the Mainnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+January 4, 2023
+
+Concordium Node 5.0.7 for MacOS
+-------------------------------
+
+A bug was fixed in the MacOS node that caused an issue with calculation of NRG.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes-mainnet.rst:19]:
```suggestion
Fix a bug in the MacOS node that caused an issue with NRG calculation.
```

I think this is more in-line with how we write changelogs.

File: source/mainnet/net/resources/release-notes.rst [hunk 2]
```diff
@@ -5,14 +5,22 @@
...
    Subscribe to the `Testnet status page <https://status.testnet.concordium.software/>`_ and the `release information on Discourse <https://support.concordium.software/c/releases/9>`_ to stay informed about updates and changes that may affect you as a node runner, including node software releases and protocol updates.
 
    To subscribe to updates on the Testnet status page click **Subscribe** to get all updates or click **Get updates** to choose to get all updates or only updates for specific products.
 
+January 4, 2023
+
+Concordium Node 5.0.7 for MacOS
+-------------------------------
+
+A bug was fixed in the MacOS node that caused an issue with calculation of NRG.
```
Comment (abizjak) [inline source/mainnet/net/resources/release-notes.rst:19]:
```suggestion
Fix a bug in the MacOS node that caused an issue with NRG calculation.
```

I think this is more in-line with how we write changelogs.

**Review by abizjak — APPROVED** [review]

**Review by concordium-cl — APPROVED** [review]
Do we want to mention that the bug caused the affected nodes to not be able to catch up? Or is it sufficient to explain that on the status page/discourse?

**Comment by concordium-cl** [comment]
> Do we want to mention that the bug caused the affected nodes to not be able to catch up? Or is it sufficient to explain that on the status page/discourse?

Let's keep it as is here. I will add it to status page and discourse instead.

**Pull request merged**